vhdbuilder/packer/convert-sig-to-classic-storage-account-blob.sh

```diff
@@ -34,6 +34,7 @@ az resource create --id $disk_resource_id  --is-full-object --location $LOCATION
 }"
 # shellcheck disable=SC2102
 sas=$(az disk grant-access --ids $disk_resource_id --duration-in-seconds 3600 --query [accessSas] -o tsv)
+echo "Converting $sig_resource_id to ${CLASSIC_BLOB}/${GEN2_CAPTURED_SIG_VERSION}.vhd"
```

Comment:
Is it duplicated since we have the same log after line 39?

Reply:
Removed.